ee/clickhouse/views/actions.py

```diff
@@ -62,12 +62,13 @@ def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
     def people(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         team = self.team
         filter = Filter(request=request)
-        shown_as = request.GET.get("shown_as")
+        entity_id = request.GET.get(ENTITY_ID)
+        entity_type = request.GET.get(ENTITY_TYPE)
 
-        if len(filter.entities) >= 1:
-            entity = filter.entities[0]
+        if entity_id and entity_type:
+            entity = Entity({"id": entity_id, "type": entity_type})
         else:
-            entity = Entity({"id": request.GET["entityId"], "type": request.GET["type"]})
+            raise ValueError("An entity must be provided for target entity to be determined")
```

Comment:
Q: Should the above lines be part of the endpoint vs part of the filter? It's leading to repetition in the two (ch and pg) endpoints. At the same time these endpoints are pretty non-standard already.

Reply:
Was thinking about it again and I added a separate util function to grab the target entity but removed it from filters altogether (the stickiness filter had it). the target entity shouldn't exist on the filter object because it's only necessary when retrieving people. The problem I was having before when trying to add the handling in the filter was that the to_dict function would break when it scanned the filter functions because it would hit `target_entity` and raise the error since a regular trend/stickiness query didn't have target_entity params

Reply:
Would a `PeopleFilter` object perhaps make sense? :)

Non-blocking.

Reply:
Yeah I think that should be done in some form for all the insights